mirage/views/draft-registration.ts

```diff
@@ -0,0 +1,22 @@
+import { HandlerContext, Schema } from 'ember-cli-mirage';
+
+export function createDraftRegistration(this: HandlerContext, schema: Schema) {
+    const attrs = this.normalizedRequestAttrs('draft-registration');
+    const now = (new Date()).toISOString();
+    const userId = schema.roots.first().currentUserId;
+    let currentUser;
+
+    if (userId) {
+        currentUser = schema.users.find(userId);
+    }
+
+    const draftRegistration = schema.draftRegistrations.create({
+        ...attrs,
+        id: `${Math.random().toString(16).slice(2, 14)}${Math.random().toString(16).slice(2, 14)}`,
```

Comment:
Might be unnecessary

Reply:
I like the realism.

Reply:
Also, we will be able to get a fake 24 digit hex number much easier when we upgrade faker:
(`faker.random.hexaDecimal(24)` is a thing in newer faker.js)